operator/docs/standard-operational-procedures/alert-loki-read-path-high-load.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Alert - LokiReadPathHighLoad"
+description: "The standard operational procedure for alert LokiReadPathHighLoad"
+lead: ""
+date: 2022-06-21T08:48:45+00:00
+lastmod: 2022-06-21T08:48:45+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "standard-operational-procedures"
+weight: 100
+toc: true
+---
+
```

Comment:
Shouldn't each page have a heading?  The `title` defined in the metadata is displayed in the navigation but doesn't appear on the page.  If users come to the page via search, it may not be clear which SOP they're looking at without a heading.